Terasology/MetalRenegades
# Add base AI characters

## Completed on upload
- [x] Adding a basic gooey character
- [x] Giving this character a basic behavior

## To be completed

Different behaviors:
- [x] Run on sight
- [x] Attack on sight
- [x] Return "home" at night (To be completed in a different PR, too large for one)

## To test
 - Launch a game with the MetalRenegades module enabled.
 - Use the command `spawnPrefab <gooeyName>`. (either friendlyGooey, neutralGooey, angryGooey, or scaredGooey)
 - The gooey should spawn where the player is looking.
